[2025-03-15T13:01:15.482+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-15T13:01:15.499+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: api_wether_dag.api manual__2025-03-15T13:01:14.521508+00:00 [queued]>
[2025-03-15T13:01:15.509+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: api_wether_dag.api manual__2025-03-15T13:01:14.521508+00:00 [queued]>
[2025-03-15T13:01:15.510+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 2
[2025-03-15T13:01:15.524+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): api> on 2025-03-15 13:01:14.521508+00:00
[2025-03-15T13:01:15.532+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=287) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-03-15T13:01:15.534+0000] {standard_task_runner.py:72} INFO - Started process 289 to run task
[2025-03-15T13:01:15.534+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'api_wether_dag', 'api', 'manual__2025-03-15T13:01:14.521508+00:00', '--job-id', '12', '--raw', '--subdir', 'DAGS_FOLDER/dag_wether.py', '--cfg-path', '/tmp/tmpdczaimhg']
[2025-03-15T13:01:15.537+0000] {standard_task_runner.py:105} INFO - Job 12: Subtask api
[2025-03-15T13:01:15.585+0000] {task_command.py:467} INFO - Running <TaskInstance: api_wether_dag.api manual__2025-03-15T13:01:14.521508+00:00 [running]> on host 157b2d7c5f40
[2025-03-15T13:01:15.842+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='api_wether_dag' AIRFLOW_CTX_TASK_ID='api' AIRFLOW_CTX_EXECUTION_DATE='2025-03-15T13:01:14.521508+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-03-15T13:01:14.521508+00:00'
[2025-03-15T13:01:15.843+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-03-15T13:01:16.041+0000] {python.py:240} INFO - Done. Returned value was: ok
[2025-03-15T13:01:16.068+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-03-15T13:01:16.069+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=api_wether_dag, task_id=api, run_id=manual__2025-03-15T13:01:14.521508+00:00, execution_date=20250315T130114, start_date=20250315T130115, end_date=20250315T130116
[2025-03-15T13:01:16.112+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-15T13:01:16.129+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-03-15T13:01:16.132+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
